Data_Structures_Questions.md

```diff
@@ -2,11 +2,11 @@ Answer the following questions for each of the data structures you implemented a
 
 ## Queue
 
-1. What is the runtime complexity of `enqueue`?
+1. What is the runtime complexity of `enqueue`? O(1)
 
-2. What is the runtime complexity of `dequeue`?
+2. What is the runtime complexity of `dequeue`? O(1)
 
-3. What is the runtime complexity of `len`?
+3. What is the runtime complexity of `len`? O(n)
```

Comment:
Well done on the top 2, but the 3rd one is actually constant time! Reason being, the counter does all the work for us, so when we call that `len` function, it's just returning self.size, which is a constant time operation!